ROADMAP.md

```diff
@@ -2,6 +2,26 @@
 
 This document defines a high level roadmap for OpenYurt development and upcoming releases. Community and contributor involvement is vital for successfully implementing all desired items for each release. We hope that the items listed below will inspire further engagement from the community to keep OpenYurt progressing and shipping exciting and valuable features.
 
+## 2025 H1
+- Support Kubernetes version 1.32.
+- Upgrade node pools to v1beta2, Support multiplexer list/watch requests at the node pool level.
```

Comment:
Support multiplexer list/watch requests at the node pool level. --> Support aggregate list/watch requests at the node pool level in order to reduce the overhead of control-plane and network traffic between cloud and edge.

Reply:
done